.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,79 @@
+name: Bug Report
+description: Something went awry
+labels: ["type: bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Need help or support?
+        Please don't open an issue! Head to https://stackoverflow.com/questions/tagged/chart.js.
+  
+  - type: markdown
+    attributes:
+      value: "Bug reports MUST be submitted with an interactive example: https://codepen.io/pen?template=BapRepQ."
+
+  - type: markdown
+    attributes:
+      value: Chart.js versions lower then 3.x are NOT supported anymore, new issues will be disregarded.
+
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: Tell us what should happen.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Current behavior
+      description: Tell us what happens instead of the expected behavior.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Reproducable sample
```

Comment:
@etimberg is this wrong, should it be `Reproducible sample`? (if so, same applies to description)

Reply:
I think you're correct, "reproducible" is the correct spelling